docs/edgeql/funcops/set.rst

```diff
@@ -394,12 +394,19 @@ Set
 ----------
 
 
-.. eql:function:: std::min(values: SET OF anytype) -> OPTIONAL anytype
+.. eql:function:: std::min(values: SET OF anyscalar) -> \
+                  OPTIONAL anyscalar
+                  std::min(values: SET OF array<anyscalar>) -> \
+                  OPTIONAL array<anyscalar>
 
     :index: aggregate
 
     Return the smallest value of the input set.
 
+    Currently, the ``min`` function does not accept :eql:type:`bool`,
```

Comment:
IMO, it's better to list what the function _does_ accept, because we are operating from a specific list of definitions in Postgres.

Reply:
In principle I can just put the entire set of polymorphic versions there, but the full list would be bigger than `to_str` [here](https://www.edgedb.com/docs/edgeql/funcops/string#function::std::to_str) (because of the flavors of the arrays).

The reason why I worded it this way is to convey the intent that we actually want to properly extend it to all orderables (at least all orderable scalars and types derived from them).